prioritymanager.sh

```diff
@@ -0,0 +1,77 @@
+#!/bin/bash
+#this script is suppose to run every 30seconds. The functions in this script change the priority of java
+#(java because snap and wings use java to run the jobs) and gdal processes. When they exceed 7 processes
+#and when they are inferior or equal to 7 processes their priority are changed, using the nice value.
+#The following was added on crontab(using root): 
+#* * * * * /home/coresyf/prioritymanager/prioritymanager.sh
+#* * * * * ( sleep 30 ; /home/coresyf/prioritymanager/prioritymanager.sh )
+
+
+#this function changes the priority of snap processes when they exceed 15 processes
+prioritySnap () {
+
+    snapIds=$(pgrep java)
+    gdalIds=$(pgrep gdal) #shows all processes by id of this app
+    snapCount=$(pgrep -c java) 
+    gdalCount=$(pgrep -c gdal) #get the number of processes running of an application
+    iterator=0
+    
```

Comment:
Only now I have understood that on the function "prioritySnap" you don't need to store "gdalIds" and "gdalCount", these variables are only required in the function "priorityGdal". Right?
Now an optional question: do you see any possible way to refactor the code and make it more simple?

Reply:
:)